cmd/build.go

```diff
@@ -38,7 +38,7 @@ func Build(ctx context.Context) *cobra.Command {
 	options := build.BuildOptions{}
 	cmd := &cobra.Command{
 		Use:   "build [PATH]",
-		Args:  utils.MaximumNArgsAccepted(1, "https://okteto.com/docs/reference/cli/#build"),
+		Args:  utils.BuildMaximumNArgsAccepted(1, "https://okteto.com/docs/reference/cli/#build"),
```

Comment:
what's `1` for? it should accept an unlimited number of args no?

Reply:
I created this func to make it compatible between versions. The new one checks if the env is enabled for manifest v2, so it will allow more than 1, while if is not enabled, it will remain only accepting 1 path

Reply:
Should we remove the Args restrictions and make this validation at the RunE function?